global.variables.yml

```diff
@@ -3,6 +3,26 @@ variables:
   ################################################## Common Variables #######################################################
   ###########################################################################################################################
 
+  # Enable or disable CARMLs CI telemetry tracking. Note: This tracks the number of deployments only.
+  # See: https://github.com/Azure/ResourceModules/wiki/The%20library%20-%20Module%20design#telemetry
+  enableDefaultTelemetry: false
+
+  ######################################
+  # Local tokens settings
+  ######################################
+
+  # the 'localToken_' prefix will be removed from the key name when the pipelines run.
+  # e.g. if you have a token in your parameter file as <<customKey>>, then the token defined in this file looks like "localToken_customKey": 'value'
+  localToken_namePrefix: 'zu3bx' # A 3-5 character length string, included in the resources names, is overridden if provided as GitHub Secret / ADO Variable
```

Comment:
In the final version to be merged what should we have here? Is it supported to have it empty?

Reply:
The token can become default as a placeholder and no longer 'carml' as we'll move to the secret version (I presume). Removing the token shouldn't impact. However not having the local one and the secret one will result in parameters not getting the namePrefix token swapped